app/assets/javascripts/application.js

```diff
@@ -16,7 +16,17 @@
 
 function ModalMarkus(elem) {
   this.modal_dialog = jQuery(elem).easyModal({
-    updateZIndexOnOpen: false
+      onOpen: function(myModal) {
+          // wait for the modal to load
+          setTimeout(function() {
+              // search for elements that can receive text as input
+              var inputs = jQuery(myModal).find("textarea, input:text");
+              if (inputs[0] != null && inputs.length > 0) {
```

Comment:
@maplecomdori Checking inputs.length > 0 should be enough; you can remove the `inputs[0] != null` check.